openmetadata-ui/src/main/resources/ui/src/components/Chart/Chart.interface.ts

```diff
@@ -11,10 +11,18 @@
  *  limitations under the License.
  */
 
+import { ColumnProfile } from 'generated/entity/data/table';
 import { MetricChartType } from '../ProfilerDashboard/profilerDashboard.interface';
 
 export interface CustomBarChartProps {
   chartCollection: MetricChartType;
   name: string;
   tickFormatter?: string;
 }
+
+export interface DataDistributionHistogramProps {
+  data: {
+    firstDayData: ColumnProfile | undefined;
+    currentDayData: ColumnProfile | undefined;
+  };
```

Comment:
We can use the `?` optional modifier for the properties instead of explicitly giving `undefined`.

```suggestion
  data: {
    firstDayData?: ColumnProfile;
    currentDayData?: ColumnProfile;
  };
```